utilities.php

```diff
@@ -872,8 +872,8 @@ public static function get_share_counts_api_status($response) {
     // Did it return at least 8 services?
     $has_majority_services = count(array_keys($response['body']['data'])) >= 8 ? true : false;
```

Comment:
Yeah we should...the reason why it will still work is because it was able to return counts for 9 services:

```
{
status: 200,
data: {
facebook: 1020725,
linkedin: 9453,
google_plus: 10606933,
pinterest: 1359571,
buffer: 8109,
stumbleupon: 14833,
reddit: 3,
vk: 1681,
yummly: 160
}
}
```